app/src/main/java/org/simple/clinic/newentry/PatientEntryScreen.kt

```diff
@@ -225,6 +226,23 @@ class PatientEntryScreen(context: Context, attrs: AttributeSet) : RelativeLayout
     }
   }
 
+  private fun setInputFieldsHint() {
+    val allInputFields: Map<Class<*>, TextInputLayout> = mapOf(
```

Comment:
Rename this to `allTextInputFields`?

Reply:
Same comment for the edit patient screen.

Reply:
Done.